src/coreclr/src/jit/gentree.h

```diff
@@ -913,6 +913,9 @@ struct GenTree
 #define GTF_SIMD12_OP               0x80000000 // GT_SIMD -- Indicates that the operands need to be handled as SIMD12
                                                //            even if they have been retyped as SIMD16.
 
+#define GTF_SIMDASHW_OP             0x80000000 // GT_HWINTRINSIC -- Indicates that the structHandle should be gotten from gtGetStructHandleForSIMD
```

Comment:
I was looking at resolving the struct handle issue (called out https://github.com/dotnet/runtime/pull/35421#issuecomment-619627448) differently at first. As far as the JIT needs to be concerned, there isn't really a difference between two `TYP_SIMD` of the same size until it needs to pass/return the value, at which point `Vector4` and `Vector128<float>` are handled differently.

However, recognizing this is non-trivial and so I went with the simpler approach of just flagging the operation as taking/returning `System.Numerics` vector types so we can call the right lookup method for the struct handle.

Unlike `GT_SIMD`, we don't need to track whether it is `SIMD12` (even if retyped to `SIMD16`) because we don't require any special codegen handling (all the relevant fixups are handled in importation) so we can just reuse the same flag value here.